src/utils/v2/fundingCycle.ts

```diff
@@ -23,8 +23,7 @@ export const hasDistributionLimit = (
     fundAccessConstraint?.distributionLimit &&
       !parseWad(fundAccessConstraint.distributionLimit).eq(
         MAX_DISTRIBUTION_LIMIT,
-      ) &&
-      fundAccessConstraint.distributionLimit !== '0',
+      ),
```

Comment:
Fixes bug where redemption rate is disabled when distribution limit is Zero (should only be disabled when distribution limit is infinite)